test/bin/manage_hypervisor_config.sh

```diff
@@ -139,6 +139,10 @@ action_create() {
 
     # Start nginx web server
     "${TESTDIR}/bin/manage_webserver.sh" "start"
+
+    "${TESTDIR}/bin/manage_prometheus.sh" "start"
+
+    "${TESTDIR}/bin/manage_loki.sh" "start"
```

Comment:
These 2 always start, in some cases we don't need to start them because there are no scenarios using Prometheus or Loki, for example: [test/scenarios/presubmits](https://github.com/openshift/microshift/tree/main/test/scenarios/presubmits)